lms/views/admin/assignment.py

```diff
@@ -25,6 +30,40 @@ def show(self):
             "assignment": assignment,
         }
 
+    @view_config(
+        route_name="admin.assignment.dashboard",
+        request_method="GET",
+        permission=Permissions.STAFF,
+    )
+    def assignment_dashboard(self):
+        assignment_id = self.request.matchdict["id_"]
+        assignment = self._get_or_404(assignment_id)
+
+        # Find instructors for the assigment
+        members = self.assignment_service.get_members(
+            assignment, role_scope=RoleScope.COURSE, role_type=RoleType.INSTRUCTOR
+        )
+        if not members:
+            raise ValueError()
+
+        lti_user = self._lti_user_service.impersonate(self.request, members[0])
+        response = HTTPFound(
+            location=self.request.route_url("dashboard.assignment", id_=assignment_id),
+        )
+        auth_token = (
```

Comment:
Maybe this should be part of lti_user_service as well, something like `as_cookie` or similar.